.github/workflows/stale.yml

```
@@ -0,0 +1,57 @@
+name: 'Close stale issues and PRs'
```

As per the tutorial, calling them "inactive" rather than "stale" is a bit clearer if you're not familiar with the terminology

```suggestion
name: 'Close inactive issues and PRs'
```